src/api.entrypoint.ts

```diff
@@ -1,24 +1,63 @@
+import "reflect-metadata"
+
 import type {Server} from "node:http"
 import type {AddressInfo} from "node:net"
 import KoaRouter from "@koa/router"
+import {ContainerBuilder} from "diod"
 import Koa from "koa"
-import {config} from "./config"
-import {firebase} from "./idp-adapters/idp-adapters"
+import {Config} from "./config"
+import {GroupsRepository} from "./database/groups-repository"
+import {
+  Implementation as GroupsImplementation,
+  createRouter as createGroupsRouter,
+} from "./generated/routes/groups"
+import {
+  Implementation as IntrospectionImplementation,
+  createRouter as createIntrospectionRouter,
+} from "./generated/routes/introspection"
+import {
+  Implementation as UsersImplementation,
```

Comment:
Having alias imports like this is annoying, and the generator should probably be adjusted to give unique export names. Additionally it makes debugging DI issues difficult as `Implementation` in logs isn't super descriptive

Reply:
solved using https://github.com/mnahkies/openapi-code-generator/pull/258